vignettes/plot_pca.Rmd

```diff
@@ -16,46 +16,48 @@ This document explains PCA, clustering, LFDA and MDS related plotting using `{gg
 
 ```{r, message = FALSE}
 library(ggfortify)
-df <- iris[c(1, 2, 3, 4)]
-autoplot(prcomp(df))
+df <- iris[1:4]
+pca_res <- prcomp(df, scale. = TRUE) # create PCA object (scale the data!)
```

Comment:
Yes，do scale before pca

Reply:
Thanks for the PR! Could you remove the comment on this line? The code itself is pretty self-explanatory.

Reply:
see my lates commit in this pull request!